daemon/list.go

```diff
@@ -516,6 +517,21 @@ func (daemon *Daemon) filterVolumes(vols []volume.Volume, filter filters.Args) (
 				continue
 			}
 		}
+		if filter.Include("label") {
+			v, err := daemon.volumes.Get(vol.Name())
```

Comment:
@cpuguy83  I did not find List() return Labels, did you try that ? From what I got in code call List(), it not have labels, also from code, I did not find it deal with Labels.

Maybe I lost something in debug, could you show to me how you get List() with labels?